### DISCUSSION TITLE
Datarange filter useing in view

### DISCUSSION THREAD

**Discussion by uzemelteteshj** [Budibase/budibase#11201] [Help]
we are using a budibase database and we have a table inside which we have created a new view. Frontend i want show datas (its work) but when i try use daterange filter i cant how can it do that

db:
![image](https://github.com/Budibase/budibase/assets/113426887/5263d268-04ae-44cd-97c0-52f2382257ca)

ui:
![image](https://github.com/Budibase/budibase/assets/113426887/ca8a7f2b-1aab-4ec9-b289-6519e71ffe35)

**Comment by melohagan** [comment]
Hey @uzemelteteshj 

Just noticing your post now! What happens when you click the `Filter` button in your view? Do you not have any options there to filter on the date?
